index.html

```diff
@@ -47,6 +47,7 @@ <h1>Popover Attribute Polyfill</h1>
           </ul>
         </div>
       </div>
+      <dialog popover>I'm a dialog!</dialog>
```

Comment:
Is this used/useful somewhere?

Reply:
Yes I used this to manually test dialogs to ensure they function as expected, given https://github.com/oddbird/popover-polyfill/pull/103/files#diff-db819e90c0d1e17da5c46e2921746208de43ceae1c1843143c47e196cef30068R23-R25